.github/workflows/linting.yml

```diff
@@ -58,3 +58,12 @@ jobs:
           ref: ${{ github.event.pull_request.head.sha }}
       - name: Check DAOS logging macro use.
         run: ./utils/cq/d_logging_check.py --github src
+
+  ftest-tags:
+    name: Ftest tag check
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
```

Comment:
If this check annotates the errors against source lines then you need to checkout the PR commit itself here or the line numbers will be wrong.  See L58 for an example of this.